pkg/router/template/util/util.go

```diff
@@ -94,3 +103,12 @@ func GenerateBackendNamePrefix(termination routev1.TLSTerminationType) string {
 
 	return prefix
 }
+
+// HaproxyMaxTimeoutDuration is HaproxyMaxTimeout as a time.Duration value.
+var HaproxyMaxTimeoutDuration = func() time.Duration {
+	d, err := time.ParseDuration(HaproxyMaxTimeout)
```

Comment:
Any reason to not use haproxtime.ParseDuration for consistency here?

Reply:
ok